site/src/_pages/docs/3_embedding-blocks.mdx

```diff
@@ -10,28 +10,62 @@ You can also [check out the source code](https://github.com/hashintel/hash/tree/
 You can browse prototype blocks in the [Block Hub](https://blockprotocol.org/hub) right now.
 You can view their schema (the structure of data they accept), and see how they look given different data.
 
-You can also [generate an API key](https://blockprotocol.org/settings/api-keys) to search our API for available blocks:
+### Block API
 
-`https://blockprotocol.org/api/blocks?q=text_query`
+You can also [generate an API key](https://blockprotocol.org/settings/api-keys) to search our API for available blocks.
```

Comment:
I feel like this should read "search the Block Protocol Hub" as opposed to "search our API". Users aren't really searching an interface, merely acting through it.

Reply:
I'll sneak this in in another PR